### ISSUE TITLE
simclr on medical dataset gives error

### ISSUE THREAD

**Issue by fatmacelebi** [google-research/simclr#187]
I just changed the line 65 in simclr/tf2/data.py (   dataset = builder.as_dataset() ) as 
dataset= tf.data.Dataset.from_tensor_slices(train_images)
but I was not able to run the codes. Is there any document that you can share with me how to apply simclr on custom dataset which different than tensorflow dataset or can you say where I am doing wrong? 
In my codes; train_images = list(paths.list_images(images_path))
images_path=folder path which contains unlabaled pretraining images
With this way I am getting error in mp_fn function with error "tf__map_fn() missing 1 required positional argument: 'label'"

**Comment by ambekarsameer96** [comment]
Hi, you can try to add labels from the dataset, that might fix the issue. You can refer to this from - 

`# Both the features and the labels tensors can be converted
# to a Dataset object separately and combined after.
features_dataset = Dataset.from_tensor_slices(features)
labels_dataset = Dataset.from_tensor_slices(labels)
dataset = Dataset.zip((features_dataset, labels_dataset))`

**Comment by fatmacelebi** [comment]
but I don't have any label, because I want to train unlabeled images as pretraining.

**Comment by chentingpc** [comment]
creating a dummy label may suffice. label won't be used anyway for the pretraining of backbone network.

**Comment by chentingpc** [comment]
These are checkpoints and you could specify
`FLAGS.checkpoint=/path/to/directory` to load the checkpoint for
fine-tuning.

**Comment by fatmacelebi** [comment]
I need Resnet model pre-trained weights to use in another approach as initial weights and I need an .h5 file to load weights. Is is possible to get such a file?

**Comment by wangxueping78** [comment]
>

> apply simclr on custom dataset

Now I want apply simclr on my custom dataset.Could you please tell me how to prepare datas format?Thank you very much!
